### ISSUE TITLE
Documentation diagrams not generated correctly

### ISSUE THREAD

**Issue by dlidstrom** [sestoft/C5#139]
### Description

The documentation that is generated by Doxygen has some kind of encoding problem in the inheritance diagrams. All characters are being shown as boxes.

### Steps to Reproduce

See <https://c5.azurewebsites.net/d2/d52/classC5_1_1GuardedEnumerator.html>. I am using Chrome on Windows but observed the same using Safari om my Macbook Pro.

![image](https://github.com/sestoft/C5/assets/371713/e3f7e3ae-adb9-41db-ba66-de260911fe02)

**Expected behavior:** [What you expect to happen]

I expect readable characters.

**Actual behavior:** [What actually happens]

Encoding problems.

Finally, maybe Doxygen isn't the best documentation generator for .NET projects? It might be a good idea to try. See [DocFX](https://dotnet.github.io/docfx/) which looks promising.

**Comment by ondfisk** [comment]
Indeed, DocFX looks promising. Will look into publishing with it on GitHub pages.
